invenio_communities/alembic/fbe746957cfc_create_member_tables.py

```diff
@@ -14,9 +14,12 @@
 
 # revision identifiers, used by Alembic.
 revision = "fbe746957cfc"
-down_revision = "f701a32e6fbe"
+down_revision = (
+    "2f63be7b7572",
+    "f701a32e6fbe",
```

Comment:
MAJOR: Question for reviewers, with this change we are able to use the upgrade command from alembic even from scratch, and I think it's correct that this dependency directly depends on the recipe that creates. (that revision id could potentially be moved to `depends_on`, to be tested if )

Reply:
To me this sounds like the "alembic merge" is the way to enforce the order we were discussing IRL. We should do that at least for the recipes requiring this explicitness, i.e. PK type changes...

Now for the rest, my gut feeling is that we should also fix them in the long term, but it looks like that's a much bigger  task... Do we know roughly how many places that is? Is there a way to incrementally fix them across modules?

Reply:
To me the "alembic merge"  is the solution, only if we need to downgrade to that recipe, or to force the path through that recipe, the `depends_on` does not enforce that path but just tells  that it needs the revision_id to be processed before being executed (meaning that many others recipes could be executed in between both recipes).

About how many places should we fix it ... So far it looks like it works with this one, however there might be other places that are affected. To find them, we should review all the recipes, to fix them incrementally, we can just fix them as they start to fail :sweat_smile:

Reply:
discussed in person, decided to go ahead with the solution and mitigate any consequences it might cause